sonar-ws/src/main/gen-java/org/sonarqube/ws/Rules.java

```diff
@@ -43,37 +43,27 @@ public static void registerAllExtensions(
    * WS api/rules/list for internal use only
    * </pre>
    */
-  public static final class ListResponse extends
+  public  static final class ListResponse extends
       com.google.protobuf.GeneratedMessage implements
       // @@protoc_insertion_point(message_implements:sonarqube.ws.rules.ListResponse)
       ListResponseOrBuilder {
     // Use ListResponse.newBuilder() to construct.
-    private ListResponse(com.google.protobuf.GeneratedMessage.Builder<?> builder) {
+    private ListResponse(com.google.protobuf.GeneratedMessage.Builder builder) {
       super(builder);
-      this.unknownFields = builder.getUnknownFields();
     }
-    private ListResponse(boolean noInit) { this.unknownFields = com.google.protobuf.UnknownFieldSet.getDefaultInstance(); }
-
-    private static final ListResponse defaultInstance;
-    public static ListResponse getDefaultInstance() {
-      return defaultInstance;
-    }
-
-    public ListResponse getDefaultInstanceForType() {
-      return defaultInstance;
+    private ListResponse() {
+      rules_ = java.util.Collections.emptyList();
     }
 
-    private final com.google.protobuf.UnknownFieldSet unknownFields;
     @java.lang.Override
     public final com.google.protobuf.UnknownFieldSet
-        getUnknownFields() {
+    getUnknownFields() {
       return this.unknownFields;
     }
     private ListResponse(
         com.google.protobuf.CodedInputStream input,
-        com.google.protobuf.ExtensionRegistryLite extensionRegistry)
-        throws com.google.protobuf.InvalidProtocolBufferException {
-      initFields();
+        com.google.protobuf.ExtensionRegistryLite extensionRegistry) {
+      this();
       int mutable_bitField0_ = 0;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00117)